<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterTabs</class>
 <widget class="QWidget" name="CharacterTabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>49</y>
     <width>281</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="statsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Look</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resistPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Resist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buffers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Misc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guild</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
